packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -577,11 +551,11 @@ const DatePickerInput = (props) => {
       return;
     }
     // decrement current valid date by 1 day, if not valid set date to yesterday instead
```

Comment:
This comment can be removed since you are decrementing date based on input field.

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1999/commits/7b61de3934e318f97325758b279207800dc84d9e